app/src/main/res/layout/activity_main.xml

```diff
@@ -1,49 +1,141 @@
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_six" />
+
+            <Button
+                android:id="@+id/button_function_minus"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_minus" />
+
+            <Button
+                android:id="@+id/button_number_one"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_one" />
+
+            <Button
+                android:id="@+id/button_number_two"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_two" />
+
+            <Button
+                android:id="@+id/button_number_three"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_three" />
+
+            <Button
+                android:id="@+id/button_function_multiplication"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_multiplication" />
+
+            <Button
+                android:id="@+id/button_number_zero"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_zero" />
+
+            <Button
+                android:id="@+id/button_function_split"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_split" />
+
+            <Button
+                android:id="@+id/button_function_erase"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_erase" />
+
+            <Button
+            android:id="@+id/button_function_result"
+            style="@style/calculatorButton"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/main_screen_btn_text_result" />
+        </GridLayout>
+    </LinearLayout>
 
 </LinearLayout>
```

Comment:
you just need one linear layout not both

Reply:
Done